### ISSUE TITLE
Menu is no more

### ISSUE THREAD

**Issue by aleksejrs** [Kaljurand/Diktofon#31]
It seems that you need to add a menu button to the UI itself.

**Comment by Kaljurand** [comment]
Yes, Diktofon still uses the old-style menu button, which is not (very) visible on newer devices. This problem has been fixed in the gradle-branch (https://github.com/Kaljurand/Diktofon/commits/gradle), but this branch is not ready yet to be merged into master.
